### ISSUE TITLE
Error while scan qrcode

### ISSUE THREAD

**Issue by wagnerfillio** [pedroslopez/whatsapp-web.js#1068]
I've already upgraded to version 1.15.3, but it always returns this error:

New Version Whatsapp: 2.2149.1

```
TimeoutError: waiting for selector `canvas` failed: timeout 45000ms exceeded
    at new WaitTask (C:\www\chatmais\backend\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:513:34)
    at DOMWorld.waitForSelectorInPage (C:\www\chatmais\backend\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:424:26)
    at Object.internalHandler.waitFor (C:\www\chatmais\backend\node_modules\puppeteer\lib\cjs\puppeteer\common\QueryHandler.js:31:77)
    at DOMWorld.waitForSelector (C:\www\chatmais\backend\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:317:29)
    at Frame.waitForSelector (C:\www\chatmais\backend\node_modules\puppeteer\lib\cjs\puppeteer\common\FrameManager.js:841:51)
    at Page.waitForSelector (C:\www\chatmais\backend\node_modules\puppeteer\lib\cjs\puppeteer\common\Page.js:2313:33)
    at Timeout.getQrCode (C:\www\chatmais\backend\node_modules\whatsapp-web.js\src\Client.js:143:28)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by PurpShell** [comment]
This issue doesn't use the bug report template that we provided, neither does it provide any information to help reproduce this issue, even if, the information is way too insignificant. Please open a new issue and don't clear what we told you to fill.
